src/main/java/org/openpnp/machine/reference/feeder/Bank.java

```diff
@@ -0,0 +1,271 @@
+/*
+
+Copyright (C) 2016 AV (CS) <phone.cri@gmail.com>
```

Comment:
If you wish to retain copyright, please remove the below license and add the standard OpenPnP license. You may leave the above copyright statement.

 * This file is part of OpenPnP.
 * 
 * OpenPnP is free software: you can redistribute it and/or modify it under the terms of the GNU
 * General Public License as published by the Free Software Foundation, either version 3 of the
 * License, or (at your option) any later version.
 * 
 * OpenPnP is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
 * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
 * Public License for more details.
 * 
 * You should have received a copy of the GNU General Public License along with OpenPnP. If not, see
 * <http://www.gnu.org/licenses/>.
 * 
 * For more information about OpenPnP visit http://openpnp.org